src/CMakeLists.txt

```diff
@@ -508,6 +508,15 @@ if (WITH_SERIALIZATION)
     target_compile_definitions(Halide PRIVATE WITH_SERIALIZATION)
 endif ()
 
+# Enable serialization testing by intercepting JIT compilation with a serialization roundtrip;
+# This is used only for special builds made specifically for testing, and must be disabled by default.
+option(WITH_SERIALIZATION_JIT "Intercepting JIT compilation with a serialization roundtrip, for test only" OFF)
```

Comment:
Suggestion: rename this to something more evocative of testing, e.g. `WITH_SERIALIZATION_JIT_ROUNDTRIP_TESTING`

Reply:
Update: I took the liberty of making this change myself, so that I could test the necessary buildbot changes